cmd/flags_misc.go

```diff
@@ -50,6 +58,18 @@ func ParseMiscFlags(cmd *cobra.Command) error {
 		logs.SetAllLoggers(level)
 	}
 
+	logModules, err := cmd.Flags().GetStringSlice(logLevelModuleFlag)
+	if err != nil {
+		return err
+	}
+	for _, ll := range logModules {
+		params := strings.Split(ll, ":")
+		err := logging.SetLogLevel(params[0], params[1])
```

Comment:
This could panic if user inputs log module the wrong way (doesn't use `:` or something). Could check that len(params) == 2.

Reply:
Yeah, true.

Reply:
Fixed